src/Command/CommandBase.php

```diff
@@ -71,38 +74,45 @@ abstract class CommandBase extends Command implements LoggerAwareInterface {
    *   An OutputInterface instance.
    *
    * @throws \Acquia\Cli\Exception\AcquiaCliException
+   * @throws \Exception
    */
   protected function initialize(InputInterface $input, OutputInterface $output) {
     $this->input = $input;
     $this->output = $output;
     $this->formatter = $this->getHelper('formatter');
     $this->setLogger(new ConsoleLogger($output));
+
+    /** @var TelemetryHelper $telemetry_helper */
```

Comment:
@danepowell I'd love to find a way to avoid needing to do this.

Reply:
Yeah I think this will all become much cleaner if/when we can inject dependencies via Symfony service config, instead of constantly polling the container.